vuepress/docs/v7.0/docs/reference/deployment-structure.md

```diff
@@ -4,154 +4,128 @@ sidebarDepth: 2
 
 
 ## entando-de-app
-The _entando-de-app_ is a J2EE application and is an instance of the _entando-core_ (see a description of the 
-_entando-core_ repo below). Reviewing the dependencies of this application in the pom.xml will reveal the 
-dependencies on the _entando-core_, _entando-engine_, and _admin-console_ which encompass the core 
-functionality in versions of Entando prior to Entando 6. In a quickstart deployment the _entando-de-app_ is deployed as part of the _entando-composite-app_ multi 
-container pod.
+The **entando-de-app** is a J2EE application and an instance of the [**entando-core**](#entando-core). It provides the server image the Entando Operator requires to manage the deployment and provide pathways for other Entando Components. The pom.xml for the application reveals its dependencies.
+
 * GitHub: <https://github.com/entando/entando-de-app/>
 * DockerHub: <https://hub.docker.com/repository/docker/entando/entando-de-app-eap>,<https://hub.docker.com/repository/docker/entando/entando-de-app-wildfly>
 
 #### Customization
-The _entando-de-app_ is very likely to be customized as part of an Entando implementation. This image can be 
-customized with new APIs, legacy Entando plugins, new database tables, or other extensions to the _entando-core_. 
-It is highly recommended that most extensions to the platform on Entando occur in microservices. However, legacy 
-integrations, extensions to the CMS, and migrations from earlier Entando versions may require changes to the _entando-de-app_. 
-
-## app-builder
-The _app-builder_ is the front end of the _entando-de-app_. It communicates with the _entando-de-app_ via [REST 
-APIs](../consume/entando-apis.md). The _app-builder_ is a React JS application and is served via node in the default 
-deployment. In a quickstart deployment, the _app-builder_ container is deployed in the _entando-app_ 
-multiple container pod. The _app-builder_ also communicates with the Component Manager via REST API to fetch 
-information about Entando bundles deployed to the Entando Component Repository (ECR).
+The **entando-de-app** is very likely to be customized as part of an Entando implementation. 
+A customized image can include:
+* new APIs 
+* legacy Entando plugins
+* new database tables  
+* other extensions to the **entando-core**. 
+
+It is highly recommended that most extensions to the Entando platform occur in microservices. However, legacy 
+integrations, extensions to the CMS, and migrations from earlier Entando versions may require changes to the **entando-de-app**. 
+
+## App Builder
+The App Builder is the user-friendly front end UI for the **entando-de-app**. A ReactJS application, it is served via node in the default deployment. In a quickstart deployment, the App Builder container is deployed in the **entando-app** multi-container pod. It communicates with the **entando-de-app** and the Entando Component Manager (ECM) via [REST 
```

Comment:
frontend

second sentence worded awkwardly -> "It is a ReactJS application served via node in the default deployment."